features/steps/partcad-cli/commands/version.py

```
@@ -0,0 +1,50 @@
+
+
+@then('command takes less than "{max_duration}" seconds')
+def step_impl(context, max_duration):
+    assert context.duration < float(max_duration)
+
+
+@then('STDOUT should contain "{substring}"')
+def step_impl(context, substring):
+    # TODO: @alexanderilyin: refactor & unify all output matching related steps
+    substring = expandvars(substring, context)
+    logging.debug("STDERR: " + context.result.stderr)
```

Ruff (0.8.0)

Logging statement uses `+`

(G003)